ParisBNF/abb/BNFabb134.xml

```diff
@@ -345,7 +335,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                      mentioned in the supplication formulas. According to
                      <bibl><ptr target="bm:Six1999VOHD3"/><citedRange unit="page">112</citedRange></bibl>,
                      the manuscript may be referred to in a book inventory included in
-                     <ref type="mss" corresp="Tanasee113">Ṭānāsee 125 Dāgā = ʾƎsṭifānos 14</ref>.
+                     <ref type="mss" corresp="Tanasee113">Ṭānāsee 125 = Dāgā ʾƎsṭifānos 14</ref>.
```

Comment:
Again: Tanasee 113 (as in `corresp`) or 125 (as in the content)?

Reply:
125

Reply:
then correct `corresp`